example/scalalib/native/1-simple/build.mill

```diff
@@ -0,0 +1,29 @@
+package build
+import mill._, scalalib._, scalanativelib._
+
+object `package` extends RootModule with ScalaNativeModule {
+  def scalaVersion = "2.13.11"
```

Comment:
Why don't you use Scala 3 ?
The example will last longer and will need fewer updates in the future if we start already with latest Scala.

Reply:
Alright.. I'll use `3.3.4`